src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/GetInstalledPackagesAsyncTelemetryEvent.cs

```diff
@@ -9,7 +9,7 @@ public sealed class GetInstalledPackagesAsyncTelemetryEvent : DiagnosticTelemetr
     {
         private const string Data = "Data";
 
-        private const string EventName = DiagnosticEventName + "/GetInstalledPackagesAsync";
+        private static readonly string EventName = $"{DiagnosticEventName}/GetInstalledPackagesAsync";
```

Comment:
You probably need to revert this, correct?

Reply:
Yes correct, Nikolche. I will revert it in the next commit.